src/watertap_contrib/reflo/costing/units/forward_osmosis_zo.py

```diff
@@ -0,0 +1,153 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2023, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+
+import pyomo.environ as pyo
+import idaes.core.util.scaling as iscale
+from watertap.costing.util import register_costing_parameter_block
+from watertap_contrib.reflo.costing.util import (
+    make_capital_cost_var,
+    make_fixed_operating_cost_var,
+)
+
+# Costing numbers are from data provided by Trevi (2021),
+# and should be applied to a Trevi FO flowsheet.
+# Mass and energy flows should be calculated from a Trevi FO flowsheet.
```

Comment:
Does this mean that this costing package is not compatible with a standalone FO flowsheet?

Reply:
Yes. The costing package only applies to the whole flowsheet and not the unit model, so the heat exchangers, pumps, etc. are not separately considered in the flowsheet.